src/store/index.js

```diff
@@ -174,6 +180,12 @@ export default new Vuex.Store({
 			let errors = [];
 			let capabilities = cx.state.connection.capabilities();
 
+			// Note down federation things from capabilities
+			let federation = capabilities.listFederation();
```

Comment:
fyi: You can obtain the object as such: `capabilities.toJSON().federation`

Reply:
Yeah but if there exists a function precisely for my use case then I kinda like to use it :D